packages/expo-gl/ios/EXGLContext.mm

```diff
@@ -39,6 +39,8 @@ - (instancetype)initWithDelegate:(id<EXGLContextDelegate>)delegate
     _isContextReady = NO;
     _wasPrepareCalled = NO;
     _appIsBackgrounded = NO;
+
+    [self initialize];
```

Comment:
I've moved this call from the view implementation to here so the setup for `glContext` lazy property is cleaner.